os/net/lwip/src/api/sockets.c

```diff
@@ -464,7 +464,7 @@ int copy_socket(void *arg)
 				sock_info->remote_port = 0;
 			}
 			if (pthread_getname_np(sockets[i].conn->pid, sock_info->pid_name)) {
-				strcpy(sock_info->pid_name, "NONE");
```

Comment:
Don't you need to add `+ 1` for NULL?

Reply:
sizeof("NONE") returns 5, including the last NULL. Therefore, it automatically copies the last NULL character, as far as I know. Thanks for the comment.